ramls/samples/profiles/jobProfile.sample

```diff
@@ -0,0 +1,29 @@
+{
+  "id": "448ae575-daec-49c1-8041-d64c8ed8e5b1",
+  "name": "job profile name",
+  "destination": "fileSystem",
+  "description": "Job profile description",
+  "userInfo": {
+    "firstName": "DIKU",
+    "lastName": "ADMINISTRATOR",
+    "userName": "diku_admin"
+  },
+  "mappingProfileId": "3798f546-7afb-11ea-bc55-0242ac130003",
+  "protocol": [
```

Comment:
Just a minor one, if we have destination as filesystem, we won't be having a protocol right